benchmarks/deconvolution-1D/server.py

```diff
@@ -0,0 +1,139 @@
+import umbridge
+import numpy as np
+import cuqi
+
+
+class Deconvolution1D(umbridge.Model):
+    """Base benchmark for all 1D deconvolution problems"""
+
+    dim = 128  # Dimension of the 1D signal
+    default_delta = 0.01  # Default value for delta
+
+    def __init__(self, name):
+        """Initialize the model
+
+        parameters
+        ----------
+        name : str
+            Name of the model
+
+        """
+        # Load data and exact phantom from file.
+        # These were originally generated by the testproblem in cuqipy,
+        # with the testproblem parameters shown below.
+        # np.random.seed was set to 0.
```

Comment:
Good that details are provided, but once again to remove any possible source of misunderstanding I think the precise call should appear somewhere, for example in a separate file (along with CUQIpy version used to generate it, operating system, seed etc)